packages/slice-machine/lib/models/common/widgets/Timestamp/Mock/Form.js

```diff
@@ -1,11 +1,12 @@
 import TimeConfigForm from "components/TimeConfigForm";
-
-import { initialValues } from ".";
+import { DefaultConfig } from "@lib/mock/LegacyMockConfig";
 
 const Form = (props) => {
-  return <TimeConfigForm {...props} initialMockValues={initialValues} />;
+  return (
+    <TimeConfigForm {...props} initialMockValues={DefaultConfig.Timestamp} />
```

Comment:
it seems it is the second time set the initial value twice.
Still unsure on why we need to do that.

Reply:
I don't get your message, can you reexplain pls? :)

Reply:
It is about having a props `initialMockValue` ➕  the `Form.initialValue = DefaultConfig.....`